bin/api-docs/gen-block-lib-list.js

```diff
@@ -176,6 +182,10 @@ const files = glob.sync(
 
 let autogen = '';
 
+autogen += `Last updated: ${
+	new Date().toISOString().split( 'T' )[ 0 ]
+}. \n\n`;
+
```

Comment:
This code does not appear to check whether the document was actually updated, as it outputs the current date and time.

The GitHub Action (Static Analysis) runs `other:check-local-changes` script. This script runs the `docs:build` script internally. Then, this script executes `bin/api-docs/gen-block-lib-list.js`. It would then update the `docs/reference-guides/core-blocks.md` file with the date and time the GitHub action was executed as the last update date and time. So unless it is the same day the GitHub action was executed, this action will fail.

I'm wondering if there is a better way to solve this 🤔

Reply:
My ideas for achieving this are as follows:

- Use `simple-git` to get the latest updated date and time of `core-blocks.md` file.
- Get the modified date in the previous commit.
- If the updated date is different, assume the document has been updated and update the "Last Uupdated:" field.

However, I am inexperienced with node scripting and would appreciate advice from anyone who is familiar with it 🙏

Reply:
I would really like for this to be included in the docs, but for this PR, maybe it is better to remove it and merge the other parts.

Reply:
Agreed. If we exclude the generation of the update date, the Static Analysis check should also pass.
We can continue to explore how to properly generate the updated date in #38573.